signingscript/exceptions.py

```diff
@@ -1,23 +1,45 @@
+"""Signingscript exceptions."""
 from scriptworker.exceptions import ScriptWorkerTaskException
 from scriptworker.constants import STATUSES
 
 
 class TaskVerificationError(ScriptWorkerTaskException):
+    """Something went wrong during task verification."""
+
     def __init__(self, msg):
+        """Initialize TaskVerificationError.
```

Comment:
For a docstring tool, it doesn't have many docs: https://gitlab.com/pycqa/flake8-docstrings :)